tests/acceptance/conftest.py

```diff
@@ -85,14 +86,23 @@ def login(arsenic):
 
 @pytest.fixture(autouse=True, scope='function')
 def cleanup_workflows_tables(app):
-    with app.app_context():
-        obj_types = (
-                WorkflowsAudit.query.all(),
-                WorkflowsPendingRecord.query.all(),
-                workflow_object_class.query(),
-        )
-        for obj_type in obj_types:
-            for obj in obj_type:
-                obj.delete()
-
-        db.session.commit()
+    obj_types = (
+            WorkflowsPendingRecord.query.all(),
+            workflow_object_class.query(),
+    )
+    for obj_type in obj_types:
+        for obj in obj_type:
+            obj.delete()
+
+    for audit in WorkflowsAudit.query.all():
+        db.session.delete(audit)
+
+    for workflow in Workflow.query.all():
+        Workflow.delete(uuid=workflow.uuid)
+
+    db.session.commit()
+
+
+@pytest.fixture(autouse=True, scope='function')
```

Comment:
Mh, using two cleanup fixtures with `autouse=True` sounds a little dangerous, as there might be some timing/ordering issues between the two...